src/coreclr/jit/emitarm64sve.cpp

```diff
@@ -4383,6 +4383,18 @@ void emitter::emitInsSve_R_R_R(instruction     ins,
         case INS_sve_ld1w:
         case INS_sve_ld1sw:
         case INS_sve_ld1d:
+        case INS_sve_ld2b:
+        case INS_sve_ld2h:
+        case INS_sve_ld2w:
+        case INS_sve_ld2d:
+        case INS_sve_ld3b:
+        case INS_sve_ld3h:
+        case INS_sve_ld3w:
+        case INS_sve_ld3d:
+        case INS_sve_ld4b:
+        case INS_sve_ld4h:
+        case INS_sve_ld4w:
+        case INS_sve_ld4d:
```

Comment:
Remind me, but why can't we directly call `emitInsSve_R_R_R_I()` from `genHWIntrinsic()`?

Reply:
I guess we could do that manually.   Wouldn't it fall under the special codegen flag?

Reply:
yes. we will have to do that anyway for asserting.